modules/schematics/src/container/files/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template

```diff
@@ -20,9 +20,8 @@ describe('<%= classify(name) %>Component', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(<%= classify(name) %>Component);
     component = fixture.componentInstance;
-    store = TestBed.get<Store>(Store);
+    store = TestBed.get(Store);
```

Comment:
We should use the generic `TestBed.get`, as this is [deprecated](https://angular.io/api/core/testing/TestBed#get)

Reply:
ah ok, I was wondering why that changed.  Fixed in 689fc93.

Doing a quick file search, looks like we have a bunch of code across the project that still uses the non-generic form.  Want me to create an issue/open a PR for it?

Reply:
That's fine by me @jtcrowson 👍